dokuwiki-syntax/src/test/resources/dokuwiki10/specific/fullsyntax/fullsyntax.test

```diff
@@ -1434,7 +1434,7 @@ endParagraph
 onMacroStandalone [code] [] [[[http://php.net|{{wiki:dokuwiki-128.png}}]]]
 beginParagraph
 beginLink [Typed = [false] Type = [url] Reference = [http://php.net]] [false]
-onWord [{{wiki:dokuwiki-128.png}}]
+onImage [Typed = [false] Type = [url] Reference = [dokuwiki-128.png]] [false]
```

Comment:
The type looks wrong to me since the source seems to be referring to an attachment. It's definitely not an URL.

Reply:
Then should it be an attachment, icon or unknown?

Reply:
I'll modify the resource type to an attachment.